tests/Filter/CallbackFilterTest.php

```diff
@@ -81,33 +81,33 @@ public function testFilterException(): void
     {
         $this->expectException(\RuntimeException::class);
```

Comment:
I know this is not part of the issue fixed in this PR, but could you please move this assertion just before the line intended to throw the exception?